foundryvtt-devMode.mjs

```diff
@@ -4,6 +4,7 @@ import { DevModeConfig } from './module/classes/DevModeConfig.mjs';
 import { _devModeDisplayUsabilityErrors } from './module/patches/displayUsabilityErrors.mjs';
 import { libWrapper } from './module/shim.mjs';
 import setupDevModeAnchor from './module/hooks/dev-mode-anchor.mjs';
+import './module/hooks/json-changes.mjs';
```

Comment:
```suggestion
import setupJSONDiff from './module/hooks/json-changes.mjs';
```

Reply:
This should then be invoked in the hook on line 59. The goal is to have one registered `ready` hook which does many things that are split into multiple files for maintainability.